src/test/java/atdd/path/web/UserAcceptanceTest.java

```diff
@@ -0,0 +1,83 @@
+package atdd.path.web;
+
+import atdd.path.AbstractAcceptanceTest;
+import atdd.path.application.dto.CreateUserRequestView;
+import atdd.path.application.dto.UserResponseView;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.MediaType;
+import reactor.core.publisher.Mono;
+
+public class UserAcceptanceTest extends AbstractAcceptanceTest {
+    public static final String BASE_URI = "/user";
```

Comment:
복수로 지정해주면 좋을 것 같아요 :)
https://restfulapi.net/resource-naming

```suggestion
    public static final String BASE_URI = "/users";
```